packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -171,13 +187,16 @@ export abstract class LibraryCommandletExecutor<T>
         channelService.showChannelOutput();
       }
 
-      if (success) {
-        notificationService
-          .showSuccessfulExecution(this.executionName)
-          .catch(e => console.error(e));
-      } else {
-        notificationService.showFailedExecution(this.executionName);
+      if (!this.cancelled) {
```

Comment:
Do we want to move this to `(progress, token) => {...}` callback body, after this.run? ex.:
```
const result = this.run(response, progress, token);
if (!token || !token.isCancellationRequested) {
  // show success / fail notification
}
return result;
```
That way we don't have to define an extra class variable.